### PR TITLE
chore(ui): add extra checks for popups in block editor

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

I worked on adding extra checks for popups in the block editor to avoid run time errors when the API is failing or data is not coming from the API.

#
### Type of change:

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/hashtag/hashtagSuggestion.ts [hunk 1]
```diff
@@ -101,25 +102,30 @@ export const hashtagSuggestion = () => ({
         if (!props.clientRect) {
           return;
         }
-
-        popup[0].setProps({
-          getReferenceClientRect:
-            props.clientRect as Props['getReferenceClientRect'],
-        });
+        if (!isEmpty(popup)) {
+          popup[0].setProps({
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/hashtag/hashtagSuggestion.ts:106]: Can we extract it to a variable and use it across?

File: openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/mention/mentionSuggestions.ts [hunk 2]
```diff
@@ -92,24 +93,30 @@ export const mentionSuggestion = () => ({
           return;
         }
 
-        popup[0].setProps({
-          getReferenceClientRect:
-            props.clientRect as Props['getReferenceClientRect'],
-        });
+        if (!isEmpty(popup)) {
+          popup[0].setProps({
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/mention/mentionSuggestions.ts:97]: same

File: openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/slashCommand/renderItems.ts [hunk 3]
```diff
@@ -50,14 +51,19 @@ const renderItems = () => {
       if (!props.clientRect) {
         return;
       }
-
-      popup[0].setProps({
-        getReferenceClientRect:
-          props.clientRect as Props['getReferenceClientRect'],
-      });
+      if (!isEmpty(popup)) {
+        popup[0].setProps({
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/BlockEditor/Extensions/slashCommand/renderItems.ts:55]: same

**Comment by Sachin-chaurasiya** [comment]
@chirag-madlani , I have addressed the comments, please review them.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**